memor/session.py

```diff
@@ -113,13 +113,13 @@ def __contains__(self, message: Union[Prompt, Response]) -> bool:
         """
         return message in self._messages
 
-    def __getitem__(self, index: int) -> Union[Prompt, Response]:
+    def __getitem__(self, identifier: Union[int, slice, str]) -> Union[Prompt, Response]:
         """
-        Return the Session message(s).
+        Get a message from the session object.
 
-        :param index: index
+        :param identifier: message identifier (index or id)
```

Comment:
change this doc-string to also reflect the `slice` type

Reply:
fix in https://github.com/openscilab/memor/pull/108/commits/c53c4827ce9608a0918dfa7b5b583b76a9332918